### PR TITLE
Auto update java rook

### PR DESCRIPTION
**Pull request by adirook**
- Adding the required steps to the Makefile for upgrading our java rook in the operator

- Maintaining external docker version inside yml file (manifest.yml)

- This pr should be merged before this one: https://github.com/Rookout/jenkins-k8s/pull/307

### CODE REVIEW

**Comment by adirook** [comment]
https://rookout.atlassian.net/browse/RK-9612

File: version_utils.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
+import sys
+import os
+from yaml import full_load, dump
+
+CMD_READ = 'read'
+CMD_BUMP = 'bump'
+VERSION_ENV_VAR = 'VERSION_TO_PUBLISH'
+
+if len(sys.argv) < 2:
```
Comment (Gilwe) [inline version_utils.py:9]: is `python version_utils.py read` not len(sys.argv) == 1?

File: Makefile [hunk 2]
```diff
@@ -159,22 +176,26 @@ publish-operator:
 	gcloud docker -- pull us.gcr.io/rookout/rookout-k8s-operator-init-container-ubi:${INNER_VERSION}
 
 	# Tagging image with dockerhub name and right version
-	docker tag us.gcr.io/rookout/rookout-k8s-operator:${INNER_VERSION} rookout/k8s-operator:${VERSION_TO_PUBLISH}
-	docker tag us.gcr.io/rookout/rookout-k8s-operator-ubi:${INNER_VERSION} rookout/k8s-operator-ubi:${VERSION_TO_PUBLISH}
+	docker tag us.gcr.io/rookout/rookout-k8s-operator:${INNER_VERSION} rookout/k8s-operator:`python version_utils.py read`
```
Comment (Gilwe) [inline Makefile:179]: so we expect the user to increase the version? what if they don't increase it in their PR?

File: version_utils.py (hunk 1 above)
Comment (adirook) [reply version_utils.py:9]: Nope because the first arg is the python file name `version_utils.py`

File: Makefile (hunk 2 above)
Comment (adirook) [reply Makefile:179]: We always take their latest version no matter what and publish new container

File: version_utils.py (hunk 1 above)
Comment (Gilwe) [reply version_utils.py:9]: welp, now i know 😅

File: Makefile (hunk 2 above)
Comment (adirook) [reply Makefile:179]: This script run by jenkins so the user don't need to do nothing

**Review by Gilwe — APPROVED** [review]

**Pull request merged**